### PR TITLE
Fix issues with migration

### PR DESCRIPTION
**Pull request by macobo**
Original: https://github.com/PostHog/posthog/pull/4513/files#diff-b730f4a66980fe3fbf4d00c30847a9b31e0d117588038fd474e2fbb7f14d5b81R11

Problems:
1. Referencing app code in migration will break as soon as app code
changes
2. Default clause did nothing on the db side

Also worth noting this was quite the expensive migration in prod,
ideally this could have been done with batches.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: posthog/migrations/0156_insight_short_id.py [hunk 1]
```diff
@@ -24,9 +24,7 @@ class Migration(migrations.Migration):
         ),
         migrations.RunPython(create_short_ids, migrations.RunPython.noop),
         migrations.AlterField(
-            model_name="dashboarditem",
-            name="short_id",
-            field=models.CharField(blank=True, max_length=12, default=posthog.models.dashboard_item.generate_short_id),
+            model_name="dashboarditem", name="short_id", field=models.CharField(blank=True, max_length=12),
```
Comment (paolodamico) [inline posthog/migrations/0156_insight_short_id.py:27]: Is the default not needed in the migration? If that's the case, we should remove lines 26-28 altogether, we're altering the field to the same state.

Comment (macobo) [reply posthog/migrations/0156_insight_short_id.py]: You're right, removed.

`default` in a migration should refer to a db default. In this case though the default is generated by the code when python tries to commit the model.

Comment (paolodamico) [reply posthog/migrations/0156_insight_short_id.py]: I think Django still expects this. Django thinks a migration is outstanding I believe?

Comment (Twixes) [reply posthog/migrations/0156_insight_short_id.py]: Paolo is right, Django needs this as even though this default is not applied to the DB, model changes in migrations in the end must always resolve to the model's current state

**Comment by macobo** [comment]
This is not needed turns out - django will complain if you ever rename the function hence no broken migrations! :)

**Pull request closed without merging**